### PR TITLE
site-nav

### PR DESCRIPTION
**Pull request by goshakri**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="Page">
       <div class="Page-header">
-        <div class="header__section">
+        <div class="Site-Navigation">
```
Comment (hsher) [inline index.html:12]: Wrong name. Read more in css bliss documentation.

File: assets/css/style.css [hunk 2]
```diff
@@ -47,7 +47,7 @@ body {
 }
 /*HELP*/
 
-.header__section {
+.Site-Navigation {
```
Comment (hsher) [inline assets/css/style.css:50]: Functionality for this css bliss module is not completed.

File: index.html [hunk 3]
```diff
@@ -18,7 +18,7 @@
             <a href="#">Store</a>
           </div>
         </div>
-        <div class="header__section">
+        <div class="Site-Navigation">
```
Comment (hsher) [inline index.html:21]: Same as above

File: index.html [hunk 4]
```diff
@@ -9,27 +9,27 @@
   <body>
     <div class="Page">
       <div class="Page-header">
-        <div class="header__section">
-          <div class="header__item"></div>
-          <div class="header__item headerButton">
+        <div class="SiteNavigation">
+          <div class="SiteNavigation-item"></div>
+          <div class="SiteNavigation-item headerButton">
```
Comment (hsher) [inline index.html:14]: What should headerButton do?

File: index.html [hunk 5]
```diff
@@ -9,27 +9,27 @@
...
+        <div class="SiteNavigation">
+          <div class="SiteNavigation-item headerButton">
             <a href="#">Gmail</a>
           </div>
-          <div class="header__item headerButton">
+          <div class="SiteNavigation-item headerButton">
             <a href="#">Images</a>
           </div>
-          <div class="header__item header__bar">
+          <div class="SiteNavigation-item header__bar">
```
Comment (hsher) [inline index.html:28]: What should header__bar do?

File: index.html [hunk 6]
```diff
@@ -9,27 +9,27 @@
...
             <a href="#"></a>
             <img src="assets/images/bar.png" height="30px" width="30px" />
           </div>
-          <div class="header__item header__join">
+          <div class="SiteNavigation-item header__join">
```
Comment (hsher) [inline index.html:32]: Not a bliss convention

**Comment by hsher** [comment]
@goshakri Add link to PR in trello in ticket

File: assets/css/style.css [hunk 7]
```diff
@@ -38,32 +38,27 @@ body {
...
 }
 
-.header__item:hover {
+.SiteNavigation-item:hover {
   text-decoration: underline;
 }
 
-/*HELP*/
-.header__bar:hover {
+.header__join {
```
Comment (hsher) [inline assets/css/style.css:45]: You don't use this class in your html page

File: index.html [hunk 8]
```diff
@@ -9,27 +9,27 @@
...
+          <div class="SiteNavigation-item">
             <a href="#">Images</a>
           </div>
-          <div class="header__item header__bar">
+          <div class="SiteNavigation-item">
             <a href="#"></a>
             <img src="assets/images/bar.png" height="30px" width="30px" />
           </div>
-          <div class="header__item header__join">
+          <div class="SiteNavigation-item--header__join">
```
Comment (hsher) [inline index.html:32]: Not a bliss convention

**Comment by goshakri** [comment]
[REFACT: Update header section to bliss](https://trello.com/c/qrbOmSQM/17-refact-update-header-section-to-bliss)

**Comment by goshakri** [comment]
[REFACT: Update header section to bliss](https://trello.com/c/qrbOmSQM/17-refact-update-header-section-to-bliss)

**Comment by goshakri** [comment]
[REFACT: Update header section to bliss](https://trello.com/c/qrbOmSQM/17-refact-update-header-section-to-bliss)

**Comment by goshakri** [comment]
[REFACT: Update header section to bliss](https://trello.com/c/qrbOmSQM/17-refact-update-header-section-to-bliss)

**Comment by goshakri** [comment]
[REFACT: Update header section to bliss](https://trello.com/c/qrbOmSQM/17-refact-update-header-section-to-bliss)

**Comment by goshakri** [comment]
[REFACT: Update header section to bliss](https://trello.com/c/qrbOmSQM/17-refact-update-header-section-to-bliss)

**Pull request merged**